From eac966a85dd022709fc43fd4131eb9399050c4f7 Mon Sep 17 00:00:00 2001
From: Joshua Yan <52796499+yanjos-dev@users.noreply.github.com>
Date: Thu, 4 Jun 2020 14:25:53 -0700
Subject: [PATCH 1/5] Initialize variable to prevent errors

Prevent warnings and errors related to
using a variable that has potentially
not been initialized yet.
---
 src/cborvalidation.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/cborvalidation.c b/src/cborvalidation.c
index 08c3511..4f63916 100644
--- a/src/cborvalidation.c
+++ b/src/cborvalidation.c
@@ -379,7 +379,7 @@ static inline CborError validate_floating_point(CborValue *it, CborType type, ui
     CborError err;
     int r;
     double val;
-    float valf;
+    float valf = 0.0;
     uint16_t valf16;
 
     if (type != CborDoubleType) {
-- 
2.22.0.windows.1

